app/main/views/suppliers.py

```diff
@@ -471,3 +481,80 @@ def update_supplier_framework_details(supplier_id, framework_slug):
         return jsonify(message="Database Error: {0}".format(e)), 400
 
     return jsonify(frameworkInterest=interest_record.serialize()), 200
+
+
+@main.route('/suppliers/<int:supplier_id>/frameworks/<framework_slug>/variation/<variation_slug>', methods=['POST'])
```

Comment:
So, if we are going with `POST`, then the URL would be `'/suppliers/<int:supplier_id>/frameworks/<framework_slug>/variation`, so that it's clear that we create a new variation. This is what we do with briefs/services/users etc. But recognising that changing that would require a lot of changes to the endpoint, maybe we should stick with PUT.